daffodil-lib/src/main/scala/org/apache/daffodil/lib/validation/XercesValidator.scala

```diff
@@ -57,7 +57,7 @@ object XercesValidatorFactory {
       if (config.hasPath(XercesValidator.name))
         config.getStringList(XercesValidator.name).asScala
       else Seq.empty
-    XercesValidator.fromFiles(schemaFiles)
+    XercesValidator.fromFiles(schemaFiles.toSeq)
```

Comment:
Ok, so what exactly are the rules about when you must have the "()" and when not?

Reply:
It turns out Scala 2.13 doesn't always complain about all nullary methods 

"For reasons of backwards compatibility, Scala 3 for the moment also auto-inserts () for nullary methods that are defined in Scala 2, or that override a method defined in Scala 2."

Which toSeq seems to fall under, shall I add it to toSeq?

https://docs.scala-lang.org/scala3/reference/dropped-features/auto-apply.html